Feat/tela meditacoes

ThaisMap opened the pull request:
Adiciona tela com lista de meditações e a tela do player 
Closes JOR-66

ThaisMap committed:
Merge branch 'feat/eventos-analytics' into feat/tela-meditacoes

ThaisMap committed:
feat: adiciona entity, repository e services de meditações

ThaisMap committed:
Merge branch 'develop' into feat/tela-meditacoes

ThaisMap committed:
feat: adiciona tela com lista de meditações

ThaisMap committed:
Merge branch 'develop' into feat/tela-meditacoes

ThaisMap committed:
feat:  adiciona componente AudioPlayer

ThaisMap committed:
feat: adiciona tela de Player para meditação

ThaisMap committed:
feat: adiciona imagem com animação

ThaisMap committed:
feat: separa animação em um componente

kellynvd commented on src/components/AudioPlayer.tsx:
```diff
@@ -0,0 +1,152 @@
+  }, [isPlaying])
+
+  useEffect(() => {
+    const buscarMeditacao = async () => {
```
aqui que poderia ser um nome mais genérico, já que o componente recebe o source como prop, o que acha?

kellynvd commented:
Ficou bem legal a animaçào do botão @ThaisMap 😍 Além do comentário acima, dois pontos só:

- [x] Quando tem o loading o Slider fica muito abaixo na tela, se for possível dar um padding pra ele nào ficar tão colado em baixo:
![IMG_5645](https://user-images.githubusercontent.com/13313889/135685836-3b8349ae-56bd-4881-963c-3dab8eb35d0e.PNG)

- [x] Ao dar play nessa meditação (ver imagem) ficou só rodando o loading e deu esse erro no console:

![Screenshot from 2021-10-01 17 21 01](https://user-images.githubusercontent.com/13313889/135685913-46c5be69-4194-4a31-b952-c54e91ec7d53.png)


Acredito que seja um problema com a meditação, mas seria bom um feedback pro usário, que nào foi posssível carregar ou algo assim,  ao invés de ficar só o loading.

ThaisMap committed:
feat: mensagem de erro e ajute do loading

- Exibe uma mensagem de erro se não conseguir carregar o áudio ou acontecer outro problema durante o playback
- Ajusta o loading para substituir todo o layout do componente, em vez de só o botão
